### PR TITLE
Add base image with enftun

### PR DESCRIPTION
**Pull request by zanebeckwith**
This PR:
- Updates the README to use the correct `enfcli iam` commands
- Adds a new server `enftun` to act as a base image
  - This image just installs `enftun` into a Debian Stretch base, and sets up the scripts/config for it.

### CODE REVIEW

File: enftun/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    rm -rf /var/lib/apt/lists/*
+
+# Install the Xaptum APT repo
+COPY xaptum.list /etc/apt/sources.list.d/xaptum.list
+
+# Install required packages
+RUN apt-get update  && \
+    apt-get install -y --no-install-recommends      \
+      enftun                                        \
+      systemd                                       \
```
Comment (drbild) [inline enftun/Dockerfile]: I don't believe that `systemd` is necessary.

**Review by drbild — CHANGES_REQUESTED** [review]
Just the one `systemd` leftover to remove, otherwise looks good.

File: enftun/Dockerfile (hunk 1 above)
Comment (zanebeckwith) [reply enftun/Dockerfile]: Ah, yep, sorry, leftover from when I was attempting to control the enftun via its systemd service.

I'll remove it.

**Review by drbild — APPROVED** [review]
LGTM.

**Pull request merged**